rds-update-rdsh-collection/azuredeploy.json

```diff
@@ -267,7 +266,7 @@
                         "typeHandlerVersion": "2.11",
                         "autoUpgradeMinorVersion": true,
                         "settings": {
-                            "ModulesUrl": "[concat(variables('assetLocation'),'/Configuration.zip')]",
+                            "ModulesUrl": "[concat(parameters('_artifactsLocation'),'/Configuration.zip')]",
```

Comment:
you'll also need a sasToken on here (see the BP doc) since storage is secure by default...